### DISCUSSION TITLE
[ALL Firebase Services] Support For Firebase with Flutter Desktop 🖥

### DISCUSSION THREAD

**Discussion by YazeedAlKhalaf** [firebase/flutterfire#5624] [Feature request] [answered]
I think of having a collaboration between flutter team and Firebase team for making this possible. Supporting desktop on Firebase which then helps Flutter desktop Platforms. What I mean is having a button for macOS, a button for Windows, a button for Linux on the main page of Firebase project. Then the Firebase team and flutter team with the help of the community can help in integrating it into Flutter desktop.

If this is possible, it will be a big move in the industry. Like imagine ability to add authentication for all platform using one codebase, it is insane and cool 😎.

@timsneath

**Comment by Salakar** [comment, marked as answer]
Closing in favour of #5557

Note: this is now planned, more on this soon.
